libnavigation-core/src/test/java/com/mapbox/navigation/core/routerefresh/RouteRefreshControllerTest.kt

```diff
@@ -40,9 +40,6 @@ import java.util.concurrent.TimeUnit
 @OptIn(ExperimentalMapboxNavigationAPI::class, ExperimentalCoroutinesApi::class)
 class RouteRefreshControllerTest {
 
-    @get:Rule
-    val coroutineRule = MainCoroutineRule()
```

Comment:
Route refresh controller doesn't use `Dispathers.Main`, so there is no need to set it for tests